atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -79,6 +79,8 @@ public long leaderPingResponseWaitMs() {
 
     @Value.Check
     protected final void check() {
+        Preconditions.checkState(quorumSize() == 1 && leaders().size() > 2,
```

Comment:
This check is already covered below (`Preconditions.checkState(quorumSize() > leaders().size() / 2,`)